legacyman_parser/parse_countries.py

```diff
@@ -98,12 +110,39 @@ def extract_nst_countries_in_region(soup: BeautifulSoup = None,
                 src = image.get('src')
                 style = image.get('style')
 
+            link_href = dirname(dirname(url))+str(href).replace("../", "/")
+            COUNTRY_TABLE_COLLECTION_LINKS.append(link_href)
+
             tr.append(TableLink(td.text, href, src, style))
         
         rows.append(tr)
         
     COUNTRY_TABLE_FOUND.append(RichCollection(title.text, body, None, url, cols, rows))
 
+def extract_collections_non_standard_country(soup: BeautifulSoup = None,
+                                parsed_url: str = None,
+                                parent_url: str = None,
+                                userland_dict: dict = None) -> []:
+
+    url = userland_dict['url']
+    title = soup.find('h2')
+    div_element = soup.find('div', id='PageLayer')
```

Comment:
sorry, we can't just look for `PageLayer`.  That can appear multiple times on the page.

We have to look for a `td` with a `colspan` of `7`. Then find that element's parent table.  That's the table we need to process.